### PR TITLE
Implement flush method for FakeMetrics

### PR DESCRIPTION
**Pull request by PlugaruT**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -99,3 +99,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# VSCode
+.vscode/
```
Comment (LuRsT) [inline .gitignore]: This should be in your own .gitignore_global, not on the project specific gitignore

File: src/striemann/test/fakes.py [hunk 2]
```diff
@@ -64,3 +64,6 @@ def incrementCounter(self, servicename, value=1, tags=[], **kwargs):
 
     def time(self, service_name, tags=[], **kwargs):
         return FakeTimer(service_name, tags, kwargs, self)
+
+    def flush(self, is_closing=False):
+        self = list()
```
Comment (LuRsT) [inline src/striemann/test/fakes.py]: This method needs tests

File: .gitignore (hunk 1 above)
Comment (PlugaruT) [reply .gitignore]: Fair enough.

**Comment by epoplavskis** [comment]
good to see you @LuRsT

**Pull request merged**